cmd/ketch/pool_remove.go

```diff
@@ -6,14 +6,22 @@ import (
 	"io"
 
 	"github.com/spf13/cobra"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
 
 	ketchv1 "github.com/shipa-corp/ketch/internal/api/v1beta1"
 )
 
-const poolRemoveHelp = `
+const (
+	poolRemoveHelp = `
 Remove an existing pool.
 `
+	skipNsRemovalMsg = "Skipping namespace removal..."
+)
+
+var (
+	printNsRemovalErr = func(out io.Writer, err error) { fmt.Fprintf(out, "%s\n%s", err, skipNsRemovalMsg) }
```

Comment:
Why do we need to store the anonymous func in a variable? Do we change need to override `printNsRemovalErr` somewhere? is it possible to make it a standard function ?